### PR TITLE
Handle all the pep8 reports

### PR DESCRIPTION
**Pull request by standage**
This PR eliminates all the pep8 reports in #1595. At first the `make pep8` target was reporting a bunch of errors for files we're not explicitly checking, but this mysteriously went away after I handled all of the files we *do* check.

Ready for review @camillescott.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -307,23 +313,27 @@ def run(self):
...
+                        "gzlib", "gzread", "gzwrite", "infback", "inffast",
+                        "inflate", "inftrees", "trees", "uncompr", "zutil"
+                    ]
+                )
         if "bz2" not in self.libraries:
             bz2cmd = ['bash', '-c', 'cd ' + BZIP2DIR + ' && make -f '
                       'Makefile-libbz2_so all']
             spawn(cmd=bz2cmd, dry_run=self.dry_run)
-            #self.extensions[0].extra_objects.extend(
+            # self.extensions[0].extra_objects.extend(
```
Comment (betatim) [inline setup.py]: delete the dead code?

File: tests/test_cython_parsing.py [hunk 2]
```diff
@@ -126,30 +125,32 @@ def test_alphabet_wrapper():
...
 def gather_paired(stream, **kw):
     itr = BrokenPairedReader(stream, **kw)
 
     x = []
     m = 0
     num = 0
     for num, is_pair, read1, read2 in itr:
-        #print(num, is_pair, read1, read2)
-        x.append((read1.name if read1 is not None else None, \
+        # print(num, is_pair, read1, read2)
```
Comment (betatim) [inline tests/test_cython_parsing.py]: delete me?

**Comment by betatim** [comment]
I'm a fan :+1:

**Comment by standage** [comment]
I think @camillescott and I handled these in parallel. I forgot to close the PR. Sorry for the extra work @betatim.

**Pull request closed without merging**